src/resources/i18n/translations/en.json

```
@@ -391,13 +391,13 @@
   "Last": "Last",
   "Latest": "Latest",
   "Latest episode": "Latest episode",
-  "Launch screen": "Launch screen",
+  "Launch screen": "Main view",
   "Leaving App": "Leaving…",
   "License": "Copylefted libre license",
   "Light Mode": "Light",
   "Limit": "Limit",
-  "Limit the number of downloaded episodes for each podcast by default": "Limit downloaded episodes for each podcast.",
-  "Limit the number of downloaded epiosdes explanation": "Limit the number of downloaded episodes from this podcast on your device. After the download limit is exceeded, the oldest episode will be automatically deleted.",
+  "Limit the number of downloaded episodes for each podcast by default": "Individual episode-limit.",
```

Wrt. #1027

```suggestion
  "Limit the number of downloaded episodes for each podcast by default": "Individual episode-limits.",
```

I am thinking either
"Number of episodes to keep"
"Limit kept episodes to"

but in any event with just the number below, so no "Limit" string.